citools/dump_failures.py

```diff
@@ -33,6 +33,11 @@ def update_results(task):
     for line in raw_log.splitlines():
         data = json.loads(line)
         if data["action"] == "suite_start":
+            # When empty, data["tests"] is a list and not a dictionary
+            # More context: https://github.com/mozilla/mozci-tools/issues/27
+            # if "tests" not in data or type(data["tests"]) is not dict:
+            if "tests" not in data or type(data["tests"]) is not dict:
```

Comment:
just removing the last line of the comment as it is the same as the code which you are executing (I have done this so many times!)  The comment is great